python/autoencoder/batchmaker.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
```

Comment:
You could move it anywhere, the python code is self-sufficient

the only link is the parameter for the python script directory, in your_segmatch_params.yaml